tests/main/snap-run-explain/expected.in

```diff
@@ -0,0 +1,79 @@
+The extra output from snap run --explain is not suitable for parsing.
```

Comment:
The output below seems to have improved a bit from what was presented in the sprint, but it still feels very inconsistent in terms of:

- Indentation
- Capitalization
- Use of lists vs. not
- Use of parenthesis vs. not
- Use of colon (:) at the end vs. not
- Use of data inline vs. below

In addition:

- The headers seem atypical (<< foo >>)
- Some syntax hints are not entirely readable (e.g. << ... snap-confine >>)
- Better line spacing and indentation >2 would likely improve readability

I think we can do a bit better still.. feel free to book a call if you'd like to go over these points in more detail or brainstorm over something.